packages/sync/src/y-webrtc/y-webrtc.ts

```diff
@@ -1,31 +1,31 @@
 // File copied as is from the y-webrtc package with only exports
```

Comment:
We might not want to convert this file, given it is copied from an external package with minor modifications